lib/main.dart

```diff
@@ -12,7 +13,7 @@ import 'package:safety/utils/routes.dart';
 
 void main() {
   WidgetsFlutterBinding.ensureInitialized();
-  // await Firebase.initializeApp();
+  Firebase.initializeApp();
```

Comment:
as per the documentation of the firebase https://firebase.flutter.dev/docs/core/usage/ await should be there, because there could be a scenario when the app is using any firebase feature(e.g. deep-link, remote config) and it's not has been initialized yet because we are not awaiting keyboard.